editor/src/components/canvas/controls/multiselect-resize-control.tsx

```diff
@@ -206,6 +208,8 @@ export class SingleSelectResizeControls extends React.Component<SingleSelectResi
     return this.props.selectedViews.map((view, index) => {
       const target = MetadataUtils.findElementByElementPath(this.props.componentMetadata, view)
       const frame = optionalMap((metadata) => metadata.globalFrame, target)
+      const targetProps =
+        target == null ? {} : this.props.allElementProps[EP.toString(target.elementPath)] ?? {}
```

Comment:
I'm not sure if this makes a difference here (in theory it shouldn't?), but we're using `view` as the element path in line 209 so we should probably use that here too

Reply:
Fixed in 3403d1aa5.